requirements-gdal2.4.txt

```diff
@@ -1,12 +1,3 @@
-lambda-proxy~=5.0
-numpy
-rasterio[s3]
-requests
-rio-cogeo
-rio-color
-rio-tiler
-rio_tiler_mosaic
-rio_tiler_mvt
-shapely
-supermercado
-pygeos
\ No newline at end of file
+gdal==2.4
+packaging
+joblib
```

Comment:
Seems like `packaging` and `joblib` should be added in the requirements file of the lambda that builds upon this layer, as opposed to including them in the layer directly.

Do you foresee needing these in _all_ services that make use of this lambda layer?

Reply:
No, in fact today in the lambda function I created this for, I removed the need for packaging and joblib. So I think I will delete those.